model/pytorch/dffml_model_pytorch/pytorch_base.py

```diff
@@ -63,6 +63,13 @@ def __post_init__(self):
                 map(self.clstype, self.classifications)
             )
 
+class LossFunctionNotFoundError(Exception):
+    '''
+        Exception raised when loss function is misspelled.
+    '''
+    def __init__(self, message="Loss function is not valid!"):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
```suggestion
```

We don't need `__init__`. It is sufficient to just pass tje error message every time. If possible , we should report what the spelling the use gave that didn't work.